### PR TITLE
Implement external supplier profile view

### PR DESCRIPTION
**Pull request by sdonk**

### CODE REVIEW

File: profile/api/helpers.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+from django.conf import settings
+
+from api_client import api_client
+
+
+def get_supplier_profile(sso_id):
+    response = api_client.buyer.retrieve_supplier(sso_id)
+    if response.status_code == http.client.NOT_FOUND:
+        return None
+    elif response.status_code == http.client.OK:
```
Comment (richtier) [inline profile/api/helpers.py]: this is fine. for future reference `if response.ok:` is more or less a shorthand for this,

Comment (richtier) [inline profile/api/helpers.py]: oh you've already used that elsewhere :)

File: profile/api/tests/test_helpers.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    mock_request = Mock(META={'Authorization': 'Bearer 123'})
+    mock_requests.get.return_value = Mock(ok=False)
+
+    token = get_sso_id_from_request(request=mock_request)
+
+    assert token is None
+    assert mock_requests.get.called_once_with(
+        'http://test.com/user-profile/v1/',
+        headers={'Authorization': 'Bearer 123'}
+    )
```
Comment (richtier) [inline profile/api/tests/test_helpers.py]: I like these assertions more than the super implicit ones we talked about the other day :)

Comment (sdonk) [inline profile/api/tests/test_helpers.py]: Me too, that's why I didn't use `mock_requests`. Too magic for my liking.

**Pull request merged**